src/countries/countries.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@nestjs/common';
+import { Countries } from 'src/core/models/countries';
+import { BaseService } from 'src/core/services/base.service';
+import { CountriesRepository } from './countries.repository';
+
+@Injectable()
+export class CountriesService  extends BaseService<Countries>{
+
+    constructor(repository : CountriesRepository){
+        super(repository)
+    }
+
+
+
+    async getAll() : Promise<Countries[]>{
```

Comment:
@cr7hardwork please reuse base service method for this, and please apply that for all methods